src/injected/ui.cpp

```diff
@@ -120,6 +120,7 @@ std::map<std::string, int> keys{
     {"mouse_clone", 0x505},
     {"mouse_destroy", 0x405},
     {"mouse_destroy_unsafe", 0x605},
+    {"reload_enabled_scripts", 0x173} // ctrl + f4 same as playlunky

```

Comment:
Doesn't playlunky use ctrl + f5 to reload? I though ctrl + f4 only showed the menu

https://github.com/spelunky-fyi/Playlunky/blob/d98f09b1bdd680437248913524dd74d7dbc1c7d2/source/playlunky/mod/mod_manager.cpp#L574

Reply:
Yes, you are right. That is my mistake, I believe I said that on Discord.

Reply:
fixed